scripts/prepare_nerfstudio.py

```diff
@@ -0,0 +1,160 @@
+"""Creates transforms.json file and resized images for running Nerfstudio from gtsfm output.
+
+This contains code from Nerfstudio for parsing data into the Nerfstudio format.
+https://github.com/nerfstudio-project/nerfstudio/blob/a121d76fd085f0fb356abf150c089c42eecbd066/nerfstudio/process_data/colmap_converter_to_nerfstudio_dataset.py#L29
+https://github.com/nerfstudio-project/nerfstudio/blob/a121d76fd085f0fb356abf150c089c42eecbd066/nerfstudio/process_data/colmap_utils.py#L16
+
+Author: Jon Womack
+"""
+import argparse
+import json
+import os
+from enum import Enum
+from pathlib import Path
+from typing import Tuple
+
+import numpy as np
+from gtsam import Rot3
+
+from gtsfm.utils import images
+from gtsfm.utils import io as io_utils
+
+
+class CameraModel(Enum):
+    """Enum for camera types."""
+
+    OPENCV = "OPENCV"
```

Comment:
just curious, would it be cleaner to import these from https://github.com/borglab/gtsfm/blob/master/thirdparty/colmap/scripts/python/read_write_model.py#L55-L67?